datasets/CMIP-winter-median-pr.data.mdx

```diff
@@ -11,7 +11,14 @@ media:
 taxonomy:
   - name: Topics
     values:
-      - EIS
+      - Precipitation
+      - Snow
+      - Climate
+      - Climate Model
+  - name: Source
+    values:
+      - NASA EIS
```

Comment:
in the dataset overview pages we have inconsistencies where provider is included as a source. Does EIS want a tag for their group in the dataset config file?

Reply:
there are currently >20 datasets in the dashboard catalog with NASA EIS Values tags, so I will not change these until we confirm if they want a tag or not.